COMET.Web.Common/Components/Login.razor.cs

```diff
@@ -64,6 +65,20 @@ public partial class Login
         /// </summary>
         public bool LoginEnabled { get; set; } = true;
 
+        /// <summary>
+        /// The dictionary of focus status from the fields, used by blazor component
+        /// </summary>
+        private Dictionary<string, bool> fieldsFocusedStatus;
+
+        /// <summary>
+        /// The dictionary of focus status from the fields
+        /// </summary>
+        public Dictionary<string, bool> FieldsFocusedStatus
```

Comment:
Can you remove the private property and just have the public ? and just put {get; private set;} ?